spec/predicates/release.md

```diff
@@ -0,0 +1,95 @@
+# Predicate type: Release
+
+Type URI: https://in-toto.io/attestation/release
+
+Version 0.1
+
+## Purpose
+
+To authoritatively link from a specific release name and version string in a registry, to the artifact names and hashes that make up that release.
+
+## Use Cases
+
+When receiving a new release version, package registries can publish a release attestation covering the artifact names and hashes that make up that release. This allows consumers of that release version to ensure the artifacts they are consuming have not been tampered with.
+
+If these release attestations are optionally published to a transparency log, package authors (or other interested parties) can monitor when a new version of a package is released.
+
+These use cases are not hypothetical; both of them are the case today for npm's [build provenance feature], which includes a [publish attestation]. Note that while npm calls this a publish attestation, calling it a release attestation better reflects that it's coming from the package registry. Publishing often refers to an author sending content to the registry, as in [PyPI's trusted publishers feature].
```

Comment:
A publish attestation seems like a better term to me, since a registry publishes artifacts, whereas users and organization release software, eg on GitHub.

Reply:
yes naming is hard. We plan a work stream for Q1 in SLSA (or supply-chain) WG to align on naming conventions across the supply-chain. I think it'd be beneficial to work together on it and circle back on this PR after we've reached a broad consensus. Wdut?

@MarkLodato and @kpk47 will have more info

Reply:
Here's an example of another publish attestation proposal https://github.com/trailofbits/homebrew-attestation/tree/main/specs/publish/v0.1, which would also benefit from broader alignment on terminology and goals for each attestation

Reply:
Yeah, I'm not sure we'll be able to come up with a one-size-fits-all term here, but based on current input I think it makes sense to stick with "release attestation".